### PR TITLE
952 csv separator

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
How do you find the use of char '/0' as a null return? I hope this comparison is not too C++ style...

### CODE REVIEW

File: src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace OSPSuite.Infrastructure.Import.Services
+{
+   public interface ICsvSeparatorSelector
+   {
+      /// <summary>
+      /// Returns the separator to be used for the reading of the .csv file
+      /// </summary>
+      char GetCsvSeparator(string fileName);
```
Comment (msevestre) [inline src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs:8]: why not char?

File: src/OSPSuite.UI/Services/CsvSeparatorSelector.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+         _applicationController = applicationController;
+      }
+
+      public char GetCsvSeparator(string fileName)
+      {
+         using (var csvSeparatorPresenter = _applicationController.Start<ICsvSeparatorSelectorPresenter>())
+         {
+            csvSeparatorPresenter.SetFileName(fileName);
+
+            return csvSeparatorPresenter.Canceled() ? '\0' : csvSeparatorPresenter.GetCsvSeparator();
```
Comment (msevestre) [inline src/OSPSuite.UI/Services/CsvSeparatorSelector.cs:22]: this char is weird, especially becuase it leaks outside of the implementation. REturn a char? and if it's null, it's not valid

**Review by msevestre — CHANGES_REQUESTED** [review]

File: src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs (hunk 1 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs:8]: I like the suggestion for nullable types. I have changed it. Thanks Michael :)

File: src/OSPSuite.UI/Services/CsvSeparatorSelector.cs (hunk 2 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.UI/Services/CsvSeparatorSelector.cs:22]: done

**Pull request merged**